core/server/themes/active.js

```diff
@@ -84,6 +92,30 @@ class ActiveTheme {
     config(key) {
         return this._config[key];
     }
+
+    mount(blogApp) {
```

Comment:
I'm going to revisit how this works, suggest merging #8197 ahead of this for now